accelerator/apis.c

```diff
@@ -9,6 +9,7 @@
 #include "apis.h"
 #include <sys/time.h>
 #include "map/mode.h"
+#include "storage/scylla_api.h"
```

Comment:
Why did you include ScyllaDB oriented header here? Can you encapsulate it?

Reply:
I'd like to move this header to `apis.h`, which includes following headers.

```
#include "accelerator/common_core.h"
#include "common/trinary/trit_tryte.h"
#include "mam/api/api.h"
#include "mam/mam/mam_channel_t_set.h"
```